client.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python
+
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from __future__ import print_function, absolute_import
```

Comment:
I think your script will fail if it's not invoked from the directory of client.py, since the cwd will not be in the PYTHONPATH.

Reply:
It works from "/" with an absolute path to the script !
(`python /Users/geffrak/code/git/github.com/shotgunsoftware/tk-framework-desktopclient/
client.py`)